### DISCUSSION TITLE
Tutorial: How to create 2d dynamic light effects

### DISCUSSION THREAD

**Discussion by CodeAndWeb** [axmolengine/axmol#1102] [Show and tell]
<a href="https://www.codeandweb.com/spriteilluminator/tutorials/how-to-use-light-effects-with-axmol-game-engine">
<img src="https://user-images.githubusercontent.com/716009/223411336-81fe0933-28e3-4742-85a3-e705781674f7.png">
</a>

Content of the tutorial:

- How to create normal mapped sprites
- How to use normal mapped sprites with a shader to create 2d light effects

Read the complete tutorial here:

[How to create 2d dynamic light effects](https://www.codeandweb.com/spriteilluminator/tutorials/how-to-use-light-effects-with-axmol-game-engine)

**Comment by thienphuoc** [comment]
Nice! Please make more tutorials rearding the axmol thanks!
